### PR TITLE
Update comment.yml

### PR DESCRIPTION
**Pull request by ruowan**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by ruowan** [comment]
@Azure/arc-arm-api-review-board

**Comment by ruowan** [comment]
https://github.com/orgs/Azure/teams/arc-arm-api-review-board pls review this PR.

**Comment by ruowan** [comment]
@Azure/arc-arm-api-review-board  hello

**Comment by raych1** [comment]
@Azure/arc-arm-api-review-board

**Comment by raych1** [comment]
@Azure/arc-arm-api-review-board , testing

**Comment by ruowan** [comment]
@Azure/arc-arm-api-review-board , testing

**Comment by ruowan** [comment]
@Azure/arc-arm-api-review-board test new

File: .github/comment.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
...
+    onLabeledComments: "Hi @${PRAuthor} and @arc-arm-api-review-board, one or more change(s) have been detected in your Arc enabled VM's or Arc enabled Server's RPs. Please review the changes and ensure that no gaps have been introduced with respect to the ARM API modeling consistency across Azure Arc and Azure Compute. For further details, see guidelines at [Consistency in ARM Modeling](https://msazure.visualstudio.com/One/_wiki/wikis/One.wiki/377428/Consistency-in-ARM-Modeling?anchor=general-design-guidance). To approve the change(s), set the label to ArcSignedOff. If you have any questions, please mail to arcboard@microsoft.com."
```
Comment (raych1) [inline .github/comment.yml:57]: Does the email account arcboard@microsoft.com need to be changed either if we would like to use github team?

**Comment by jianyunt** [comment]
testing email notification -1

**Comment by jianyunt** [comment]
@jianyunt testing email notifcation2

**Comment by jianyunt** [comment]
@jianyunt testing email notifcation3

**Comment by jianyunt** [comment]
@Azure/arc-arm-api-review-board , testing email notification -2

**Pull request closed without merging**